tests/test_core.py

```diff
@@ -4447,6 +4442,7 @@ def test_dylink_load_compiled_side_module(self):
     self.set_setting('FORCE_FILESYSTEM')
     self.emcc_args.append('-lnodefs.js')
     self.set_setting('INITIAL_MEMORY', '64mb')
+    self.set_setting('NO_AUTOLOAD_DYLIBS')
```

Comment:
maybe a comment here? it's not clear to me why this specific test needs this setting.